hyperspy/io.py

```diff
@@ -401,18 +401,21 @@ def load_with_reader(filename, reader, signal_type=None, convert_units=False,
     return objects
 
 
-def assign_signal_subclass(dtype,
-                           signal_dimension,
-                           signal_type="",
-                           lazy=False):
-    """Given record_by and signal_type return the matching Signal subclass.
+def assign_signal_subclass(dtype, signal_dimension, signal_type="", lazy=False):
+    """Given record_by and signal_type, return the matching Signal subclass.
```

Comment:
Now that you are at it you could replace ``record_by`` with ``signal_dimension`` in the docstring.

Reply:
Done!